CMakeLists.txt

```diff
@@ -911,21 +941,24 @@ else()
         )
 endif()        
 
-# install python modules
+# install python modules and system libs
 if (PYTHON_COPY)
-    install(DIRECTORY build_tree/${PYTHON_COPY_LIB_TARGET} 
-            DESTINATION lib
-            PATTERN *.dll EXCLUDE
-            PATTERN *.a EXCLUDE
-            PATTERN *.so EXCLUDE
-            PATTERN */dist-packages EXCLUDE
-            PATTERN *.py EXCLUDE)
+    # install all sub folders of Python
+    foreach(PY_SUB_PATH ${PY_SUB_PATHS})
```

Comment:
Python's sys.path contains several paths we must copy in order to assure Python's correct functionality. We decided to remove part ```/usr``` from paths to make them shorter. sys.path: 
```
/usr/lib64/python27.zip
/usr/lib64/python2.7
/usr/lib64/python2.7/plat-linux2
/usr/lib64/python2.7/lib-tk
/usr/lib64/python2.7/lib-old
/usr/lib64/python2.7/lib-dynload
/usr/lib64/python2.7/site-packages
/usr/lib/python2.7/site-packages
```

if we remove prefix we get:
```
/lib64/python27.zip
/lib64/python2.7
/lib64/python2.7/plat-linux2
/lib64/python2.7/lib-tk
/lib64/python2.7/lib-old
/lib64/python2.7/lib-dynload
/lib64/python2.7/site-packages
/lib/python2.7/site-packages
```
thus we want to copy folders ```/lib64``` and ```/lib```. Variable ```PY_SUB_PATHS``` contains these values and it comes from Python command called in PYTHON_COPY phase